src/python/google_cloud_utils/compute_engine_projects.py

```diff
@@ -58,7 +58,8 @@ def _process_instance_template(instance_template):
   # Load file metadata
   items = instance_template['properties']['metadata']['items']
   for item in items:
-    if item['value'].startswith(FILE_SCHEME):
+    if (isinstance(item['value'], basestring) and
```

Comment:
Could you add a comment explaining this here or a potential FIXME to clean this function up a bit more? This seems fine for an exception fix but the function is a bit confusing.

Reply:
Done. Added more detailed comments to explain purpose, but a FIXME is not required.